index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css"> 
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" 
+  integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" 
+  crossorigin="anonymous">
+  <title>Product Landing Page</title>
+</head>
+<body>
+
+  <header id="header">
+    <div class="logo">
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png">
+    </div>
+
+    <nav id="nav-bar">
+      <ul class="navbar">
+        <li><a class="nav-link" href="#features">Features</a></li>
+        <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
+        <li><a class="nav-link" href="#pricing">Pricing</a></li>
+      </ul>
+    </nav>
+  </header>
+
+    <section class="presentation">
+      <p>Handcrafted, home-made masterpieces</p>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
```

Comment:
Kindly use the complete format of a form's submit action by adding the form's endpoint: <form action="https://formsubmit.co/your@email.com" method=" POST">

Reply:
Kindly use the complete format of a form's submit action by adding the form's endpoint because the  action attribute specifies where to send the form-data when a form is submitted.